multinet/api.py

```diff
@@ -65,6 +66,25 @@ def create_workspace(workspace):
     return workspace
 
 
+@bp.route("/workspace/<workspace>/aql", methods=["POST"])
+def aql(workspace):
+    """Perform an AQL query in the given workspace."""
+    query = request.data.decode("utf8")
```

Comment:
I meant to ask you about this: I tried using webargs and couldn't figure out how to get it to inject the entire body as raw data into a kwarg. If you can figure this out, it would make a great improvement here.

Reply:
Oh duh, that's my bad. I for some reason thought you were actually grabbing an argument here. Disregard the first comment, I think what you're doing is correct. However I still think adding a check that this variable isn't empty is good, as when I passed nothing in the body, the server threw an error.

Reply:
Thanks for clearing that up.

I added a check for an empty body, but I think eventually we'll rip that out when we do more comprehensive error handling for the low-level AQL query engine as part of #136.